src/modules/LanItems.rb

```diff
@@ -540,8 +540,10 @@ def ReplaceItemUdev(replace_key, new_key, new_val)
     # Returns new name
     def SetItemName( itemId, name)
       if name && !name.empty?
-        updated_rule = update_udev_rule_key(GetItemUdevRule(itemId), "NAME", name)
-        @Items[itemId]["udev"]["net"] = updated_rule
+        if @Items[itemId]["udev"]
```

Comment:
Shouldn't we instead ensure that `@Items[itemId]` has a `udev` key whose value is a hash?